data-prepper-core/src/main/java/org/opensearch/dataprepper/pipeline/server/DataPrepperServer.java

```diff
@@ -42,6 +45,7 @@ public void start() {
      * Stop the DataPrepperServer
      */
     public void stop() {
+        EXECUTOR_SERVICE.shutdownNow();
```

Comment:
Shouldn't this happen after the `stop` call?

Reply:
Good call